pkg/operation/botanist/health_check.go

```diff
@@ -671,12 +690,23 @@ func (b *Botanist) checkSystemComponents(
 	if exitCondition := checker.CheckExtensionCondition(condition, extensionConditions); exitCondition != nil {
 		return exitCondition, nil
 	}
-	if established, err := b.CheckVPNConnection(ctx, logrus.NewEntry(logger.NewNopLogger())); err != nil || !established {
-		msg := "VPN connection has not been established"
+
+	gardenerVersionGeq140, err := versionutils.CheckVersionMeetsConstraint(gardenerVersion, ">= v1.4.0")
```

Comment:
Sorry, now that 1.4 just got released yesterday you have to update to 1.5 ... :-)